docs/src/layouts/AppHeader.tsx

```diff
@@ -40,8 +42,11 @@ export default function AppHeader() {
   const changeTheme = useChangeTheme();
   const [mode, setMode] = React.useState<string | null>(null);
   const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
-
   const t = useTranslate();
+  // convert current path to corresponding repository path
+  const repoPath = pathnameToLanguage(useRouter().pathname).canonicalPathname;
+  // url is two elements: github.com/mui(organisation) + path of repository
+  const repoUrl = 'https://github.com/mui' + repoPath;
```

Comment:
```suggestion
```

Having the full GitHub repositories URLs in map might be much better for maintenance.

Reply:
Ohk I'll do that instead